lib/gui/modules/drive-scanner.js

```diff
@@ -34,16 +34,17 @@ const driveScanner = angular.module(MODULE_NAME, [
 
 driveScanner.factory('DriveScannerService', (SettingsModel) => {
   const DRIVE_SCANNER_INTERVAL_MS = 2000;
+  const FIRST_SCAN_DELAY_MS = 0;
```

Comment:
It would be nice if this was also prefixed with `DRIVE_SCANNER_` to match the other constant here.